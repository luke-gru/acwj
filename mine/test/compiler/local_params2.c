void printsumall(int a, int b, int c, int d, int e, int f, int g) {
    int h;
    printint(g);
    h = 100;
    printint(h);
    printint(a+b+c+d+e+f+g);
}

int main() {
    printsumall(1, 2, 3, 4, 5, 6, 7);
    return (0);
}
