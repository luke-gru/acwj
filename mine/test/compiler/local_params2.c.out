7
100
28
